android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -44,8 +45,12 @@
     public static final String KEY_TEXT_REPLY = "CAN_REPLY";
     public static final String NOTIFICATION_REPLIED_EVENT_NAME = "notificationReplied";
 
-    private static LinkedHashMap<String,Integer> channelIdToNotificationCount = new LinkedHashMap<String,Integer>();
-    private static LinkedHashMap<String,List<Bundle>> channelIdToNotification = new LinkedHashMap<String,List<Bundle>>();
+    private static final String PUSH_TYPE_MESSAGE = "message";
+    private static final String PUSH_TYPE_CLEAR = "clear";
+    private static final String PUSH_TYPE_UPDATE_BADGE = "update_badge";
+
+    private static Map<String, Integer> channelIdToNotificationCount = new HashMap<String, Integer>();
```

Comment:
What’s the difference between the two and why the change?

Reply:
It's just changing the type of the variables to use the `Map` interface instead of a concrete type. It's just a common practice to do it that way in Java.